app/src/main/java/com/cmput301w23t09/qrhunter/map/MapController.java

```diff
@@ -0,0 +1,3 @@
+package com.cmput301w23t09.qrhunter.map;
+
+public class MapController {}
```

Comment:
As mentioned in VC, if you plan on using this please move the methods needed from MapFragment to this class.